core/src/main/java/com/linecorp/armeria/server/AbstractHttpResponseHandler.java

```diff
@@ -75,6 +75,7 @@ boolean isDone() {
 
     void disconnectWhenFinished() {
```

Comment:
Would it make sense to remove this method and call `responseEncoder.keepAliveHandler().disconnectWhenFinished();` directly?
We can probably integrate the closing logic in `HttpServerHandler` with this in one place.

Reply:
Let me clean up the closing logic in the following work, perhaps, after merging `WebSocketService` PR.

Reply:
Thanks a lot! 🙇